lib/documentLoader.js

```diff
@@ -24,5 +48,21 @@ api.extendContextLoader = documentLoader => {
 };
 
 api.strictDocumentLoader = api.extendContextLoader(url => {
+  /**
+   * strictDocumentLoader if the url for a context is in the context map
+   * this returns a copy of an in-memory, immutable context document
```

Comment:
We should just define this in terms of `extendContextLoader` -- so we have the definition in one place. We can continue to say that the reason you'd use this method is if you don't want to load anything other than the built-in contexts provided by jsonld-signatures.